src/v/ssx/metrics.h

```diff
@@ -33,21 +38,18 @@ const auto public_metrics_handle = ss::metrics::default_handle() + 1;
 //
```

Comment:
Now that this is a base object, I think this comment needs to be updated?

Reply:
Yes fixed the references in that comment.